modules/bpi/bpi.module

```diff
@@ -318,14 +319,17 @@ function bpi_form_node_form_alter(&$form, &$form_state, $form_id) {
 
     // Guessing where to put 'body' text in the form.
     $bpi_body = isset($bpi_content['body']) ? $bpi_content['body'] : '';
-    if (isset($form[$body][$current_language][0]['value'])) {
-      $form[$body][$current_language][0]['value']['#default_value'] = $bpi_body;
-    }
-    elseif (isset($form[$body][$current_language][0])) {
-      $form[$body][$current_language][0]['#default_value'] = $bpi_body;
-    }
 
-    if ($bpi_id) {
+    $syndicated_images = bpi_get_syndicated_images();
+    if (is_array($syndicated_images)) {
```

Comment:
There might not be a better solution. 

To be honest I was also confused by lines L332 + L341:

- L332: This is the state when we are syndicating images. During this we set the syndication flag to FALSE.
- L341: When the syndication flag is set to FALSE we are in the state that no images were syndicated?